Cargo.toml

```diff
@@ -14,24 +14,24 @@ crate-type = ["cdylib"]
 
 [dependencies]
 async-trait = "0.1.77"
-c2pa = { version = "0.31.2", features = [
-	"fetch_remote_manifests",
-	"file_io",
-	"xmp_write",
+c2pa = { git = "https://github.com/contentauth/c2pa-rs", branch = "dyro/CAI-6506", features = [
```

Comment:
Let's hold off on releasing until I get this change in main. It's already been approved. Will let you know when I've created a release!

Reply:
Sounds good!

Reply:
As the version is released, I updated this to latest release.